serverAddons/PRA3/addons/PRA3_Server/Revive/fn_clientInit.sqf

```diff
@@ -15,33 +15,76 @@
 */
 [QGVAR(Settings), missionConfigFile >> "PRA3" >> "CfgRevive"] call CFUNC(loadSettings);
 
-GVAR(selections) = ["", "head", "body", "hand_l", "hand_r", "leg_l", "leg_r"];
+["playerChanged", {
+    (_this select 0) params ["_newPlayer"];
+    _newPlayer addEventHandler ["HandleDamage", FUNC(damageHandler)];
 
-// HitEffects
-call FUNC(blood);
-call FUNC(fatigue);
-call FUNC(unconscious);
+}] call CFUNC(addEventhandler);
 
-// Treatments
-call FUNC(bandage);
-call FUNC(heal);
-call FUNC(revive);
+["Respawn", {
+    (_this select 0) params ["_newUnit"];
 
-[QGVAR(Killed), {
-    (_this select 0) params ["_unit"];
+    _newUnit setVariable [QGVAR(bleedingRate), 0];
+    _newUnit setVariable [QGVAR(bloodLevel), 1];
+    _newUnit setVariable [QGVAR(isUnconscious), false, true];
+}] call CFUNC(addEventHandler);
 
-    _unit setVariable [QGVAR(cachedDamage), GVAR(selections) apply {[0]}];
-    [_unit, QGVAR(bloodLoss), 0] call CFUNC(setVariablePublic);
+[
+    {"Unload wounded person"},
+    ["LandVehicle", "Air", "Ship"],
+    3,
+    {
+        call {
+            scopeName "ActionCondition";
```

Comment:
i dont mean to remove the call. i only mean to use breakOut and not breakTo so we dont need to save the value in a variable.